scripts/o3de/o3de/github_utils.py

```diff
@@ -13,18 +13,19 @@
 import logging
 import pathlib
 import urllib.parse
+from urllib.parse import ParseResult
 import urllib.request
 import subprocess
 
-from o3de import gitproviderinterface
+from o3de import gitproviderinterface, utils
 
 LOG_FORMAT = '[%(levelname)s] %(name)s: %(message)s'
 
 logger = logging.getLogger('o3de.github_utils')
 logging.basicConfig(format=LOG_FORMAT)
 
 class GitHubProvider(gitproviderinterface.GitProviderInterface):
-    def get_specific_file_uri(parsed_uri):
+    def get_specific_file_uri(parsed_uri:ParseResult):
```

Comment:
nitpick: missing whitespace
```suggestion
    def get_specific_file_uri(parsed_uri: ParseResult):
```